src/main/java/net/sf/rails/game/specific/_1837/StockRound_1837.java

```diff
@@ -208,7 +208,7 @@ protected boolean mergeCompanies(PublicCompany minor, PublicCompany major) {
                 setSellObligationLifted(major);
             }
 
-            setPriority();
+            setPriority("MergeCompany");
```

Comment:
Maybe replace with an enum?

Reply:
Quicker to be handled than the string mechanics ?

Reply:
What do you mean by `quicker`? In my opinion it is more secure. When you use a `String` you have the possible issue of a misspelling, enums don't allow for such errors.

In theory it should also be possible to not do these checks at all by using a class comparison which is really ugly...